andela/pippin-ah-frontend
# [Feature #162727532]: Highlight and Comment on Article Text

#### What does this PR do?
 - It ensures an authenticated can highlight and comment on an article text

#### Description of Task to be completed?
- ensure a user can highlight and comment on article text

#### How should this be manually tested?
- Login and navigate to an article. 
- Select any text and click on the comment icon
- Add the comment in the comment box and click post
- The text should be highlighted
- Hover on the highlight to see your comment

#### What are the relevant pivotal tracker stories?
- [#162727532](https://www.pivotaltracker.com/story/show/162727532)
